feature-libs/customer-ticketing/root/facade/customer-ticketing.facade.ts

```diff
@@ -64,7 +64,11 @@ export abstract class CustomerTicketingFacade {
 
   abstract createTicketEvent(ticketEvent: TicketEvent): Observable<TicketEvent>;
 
-  abstract uploadAttachment(file: File, eventCode: string): Observable<unknown>;
+  abstract uploadAttachment(
+    file: File | null,
```

Comment:
same here, file shouldn't be null

Reply:
Fixed https://github.com/SAP/spartacus/pull/16383/commits/67f773742f3c238689ded8a66069c9fafa878ebc